### PR TITLE
Add linting target for makefile

### PR DESCRIPTION
**Pull request by colescott**

### CODE REVIEW

File: .astylerc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-A3jPs4NHxC150xLDCW1
+# Don't even ask.
```
Comment (erichkleung) [inline .astylerc:2]: yolo

File: README.md [hunk 2]
```diff
@@ -18,16 +19,3 @@ Team 4159 CardinalBotics robot code for 2017 FRC season
 - New source files will be automatically added to the build as long as they are in `src`
 - New libraries should be added to `sources.mk`
 - All changes require a new branch and to submit a pull request
-
-## Contributors:
```
Comment (erichkleung) [inline README.md]: lawl

**Review by erichkleung — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (colescott) [reply README.md]: oh boy

Comment (colescott) [reply README.md]: fixed

**Pull request merged**